src/ActionsSimplificationModal.js

```diff
@@ -0,0 +1,186 @@
+
+import './ActionsSimplificationModal.scss';
+
+type ActionsSimplificationModalProps = {
+    intl: IntlShape,
+    onCancel: () => void,
+    onConfirm: (allowedActions: ActionToggleRegister) => void,
+    // TODO: Flesh this definition out.
+    menuItems: {},
+    programSequence: ProgramSequence,
+    allowedActions: ActionToggleRegister,
+    show: boolean
+};
+
+type ActionsSimplificationModalState = {
+    allowedActions: ActionToggleRegister
+}
+
+class ActionsSimplificationModal extends React.Component<ActionsSimplificationModalProps, ActionsSimplificationModalState> {
+    static defaultProps = {
+        menuItems: {
+            forward1: {
+                isAllowed: true,
+                labelKey: "Command.forward1"
+            },
+            forward2: {
+                isAllowed: true,
+                labelKey: "Command.forward2"
+            },
+            forward3: {
+                isAllowed: true,
+                labelKey: "Command.forward3"
+            },
+            backward1: {
+                isAllowed: true,
+                labelKey: "Command.backward1"
+            },
+            backward2: {
+                isAllowed: true,
+                labelKey: "Command.backward2"
+            },
+            backward3: {
+                isAllowed: true,
+                labelKey: "Command.backward3"
+            },
+            left45: {
+                isAllowed: true,
+                labelKey: "Command.left45"
+            },
+            left90: {
+                isAllowed: true,
+                labelKey: "Command.left90"
+            },
+            left180: {
+                isAllowed: true,
+                labelKey: "Command.left180"
+            },
+            right45: {
+                isAllowed: true,
+                labelKey: "Command.right45"
+            },
+            right90: {
+                isAllowed: true,
+                labelKey: "Command.right90"
+            },
+            right180: {
+                isAllowed: true,
+                labelKey: "Command.right180"
+            }
+        }
+    }
+
+    constructor(props: ActionsSimplificationModalProps) {
+        super(props);
+        this.state = {
+            allowedActions: this.props.allowedActions
+        }
+    }
+
+    render() {
+        const cancelButtonProperties = {
+            label: this.props.intl.formatMessage({ id: 'ActionsSimplificationModal.cancel'},),
+            isPrimary: false,
+            onClick: this.handleClickCancel
+        };
+        const saveButtonProperties = {
+            id: 'ActionSimplificationModal-done',
+            isPrimary: true,
+            label: this.props.intl.formatMessage({ id: 'ActionsSimplificationModal.save'},),
+            onClick: this.saveChanges
+        };
+        return (
+            <ModalWithFooter
+                show={this.props.show}
+                focusOnOpenSelector={'#ActionSimplificationModal-done'}
```

Comment:
I think setting focus on the first item in the list might be better experience. Since we have unique id for each item, we could use the id to set focus.

Reply:
I tried this but it intermittently focuses without indicating anything visually.